core/src/test/java/com/adobe/aem/dot/common/analyzer/FileResolverTest.java

```diff
@@ -104,7 +104,34 @@ public void resolveOtherWildCardFileTest() {
 
     files = fileResolver.resolveFiles(realPath + File.separator + "FileResolverTest.class[2]", realBase);
     assertNotNull("Should not be null", files);
-    assertTrue("files have some files", files.size() > 0);
+    assertTrue("files should have some files", files.size() > 0);
     assertTrue("files contain this class", files.toString().contains(this.getClass().getSimpleName()));
   }
+
+  @Test
+  public void denyDirectoryPathTest() {
+    List<File> files = fileResolver.resolveFiles(realPath, realBase);
+    assertNotNull("Should not be null", files);
+    assertTrue("files should NOT have some files", files.isEmpty());
+
+    List<ILoggingEvent> logsList = listAppender.list;
+    // Check that the 'unclosed' regex was logged - "regex(.*" - should be the first entry in the logs.
+    assertTrue(logsList.get(0).getMessage().startsWith("Cannot include a directory.  Use wildcards to include the contents.  Path="));
+    assertTrue(logsList.get(0).getMessage().startsWith("Cannot include a directory.  Use wildcards to include the contents.  Path="));
+    assertEquals("Severity should be WARN.", Level.ERROR, logsList.get(0).getLevel());
+  }
+
+  @Test
+  public void allowDirectoryPathTest() {
+    String testDirectory = PathUtil.appendPaths(realPath, "rules");
+    FileResolver fileResolverAllow = new FileResolver(realBase, true);
+    List<File> files = fileResolverAllow.resolveFiles(testDirectory, realBase);
+    assertNotNull("Should not be null", files);
+    assertTrue("files should have some files", files.size() > 0);
```

Comment:
Should we check that a specific number of files were found?

Reply:
If someone adds a new rule test, it would break this test which would be a little strange.  I think finding any files kinda proves its point, the paths to directories work.